Sources/FirebaseFirestore/FirestoreErrorCode.swift

```diff
@@ -1,14 +1,34 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
-public struct FirestoreErrorCode: Error {
-  public typealias Code = firebase.firestore.Error
+@_exported
+import firebase
+@_spi(FirebaseInternal)
+import FirebaseCore
 
-  public let code: Code
+public struct FirestoreErrorCode: RawRepresentable, Error {
+  public typealias RawValue = Int
+
+  public let rawValue: Int
   public let localizedDescription: String
 
+  public init(rawValue: Int) {
+    self.rawValue = rawValue
+    localizedDescription = "\(rawValue)"
+  }
+
+  init(_ params: (code: Int32, message: String)) {
```

Comment:
Similar to `AuthError` - what is the ACL here?

Reply:
`internal`

Reply:
Done